drivers/ethernet/eth_sam_gmac.c

```diff
@@ -1973,7 +1973,9 @@ static void eth0_iface_init(struct net_if *iface)
 	}
```

Comment:
Please adjust the commit subject a bit, for example like this
`drivers: eth: gmac: Fix ethernet startup when fixed-link`

Reply:
Fixed the title of commit.  I also fixed the lint error